rabbitmq/datadog_checks/rabbitmq/rabbitmq.py

```diff
@@ -200,6 +200,50 @@ def _get_config(self, instance):
 
         return base_url, max_detailed, specified, custom_tags, suppress_warning
 
+    def _collect_metadata(self, base_url):
+        # Versioning in RabbitMQ seems to be standard semantic versioning.
```

Comment:
No need for such a long comment, just somthing on the lines of:
```
# overview endpoint returns a json with rabbit version in rabbitmq_version field.
# Rabbit versions follow semantic versioning https://www.rabbitmq.com/changelog.html
```

Reply:
Ok, I made the change here: https://github.com/DataDog/integrations-core/pull/4918/commits/6a038dd1cf237d9071cb11a794773454341759a6